tests/integration/launch_container.sh

```diff
@@ -26,7 +26,7 @@ if [[ "$platform" == *"cu1"* ]]; then # if the platform has cuda capabilities
 elif [[ "$platform" == *"deepspeed"* ]]; then # Runs multi-gpu
   runtime="nvidia"
   is_llm=true
-  shm="2gb"
+  shm="12gb"
```

Comment:
Why need 12gb? Does SM support this size?

Reply:
vLLM test requires minimum 10GB shared memory spaces

Reply:
for the best performance